sdk/highlight-ruby/highlight/lib/highlight.rb

```diff
@@ -29,7 +29,6 @@ def initialize(project_id, otlp_endpoint=OTLP_HTTP)
             @otlp_endpoint = otlp_endpoint
 
             OpenTelemetry::SDK.configure do |c|
-                c.service_name = "highlight-sdk"
```

Comment:
Will cut a new version of the ruby SDK when support is added to set service name by the customer.